css/layout.css

```diff
@@ -16,13 +16,19 @@
 .column-full,
 .column-half {
   width: 100%;
+  display: flex;
+  flex-direction: column;
```

Comment:
You should make this a separate class. You don't want to be left with all of your `column-full` elements being a column flex container.